README.md

```diff
@@ -0,0 +1,34 @@
+# Horondi
+Backend API for the `Hondri` online store
+## Rules and guidelines
+- File naming
+    - Files should be name in format `some-component.type.js`
+- Architecture
+    - Logic is separated in layers
+        - resolver layer (handles graphql actions)
+        - service layer (handles business logic, interactions with database)
+        - model layer (mapping collections to mongoose models)
+    - All business logic (any database operations or validation related to the business rules) and interaction with models is located inside service
+    - Each model should live in its own folder
+    - In each folder files should be named in format `{model-name}.{type}.js`
```

Comment:
{type} is not clear enough. Consider adding example (`{model-name}.service.js or `{model-name}.resolver.js)